app/src/main/java/com/jedsada/dangerplayground/UserData.kt

```diff
@@ -0,0 +1,8 @@
+package com.jedsada.dangerplayground
+
+data class UserData(val username : String?=null,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/686c248955664b57d570b950e7b8214cefc34d69/app/src/main/java/com/jedsada/dangerplayground/UserData.kt#L3"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>